dist/aurelia-dependency-injection.d.ts

```diff
@@ -1,185 +1,209 @@
+export interface DependencyCtor<TBase, TImpl extends TBase, TArgs = any> {
```

Comment:
Shouldn't this be `TArgs = any[]`

Reply:
That would make `new(...args: TArgs[])` an array of arrays.

Reply:
which would make us change it to `(...args: TArgs)`? Or am I understanding incorrectly that you don't want to infer the list of arguments?
repl: (check for type of `a`)
  https://www.typescriptlang.org/play/index.html#code/GYVwdgxgLglg9mABAEzgZSiYwA8AVRAUwA8pCxkBnRAQzAE8BtAXUQF5aGWA+ACgDpBNAE4BzSgC5EeAJSIA3gChEKxMMKZhSEeMUBfRYoBuI2uxTpM2XgEYANIgDkNRzKA

Reply:
I was under the impression order didn't matter. If they do, then this would be the correct description. Otherwise it would be `(TypeA|TypeB|TypeC)[]`